component-classes/index.js

```diff
@@ -383,15 +383,19 @@ export const alert = {
 };
 
 export const input = {
-  default: 'block text-m mb-0 leading-m s-text s-bg s-border hover:s-border-hover active:s-border-selected rounded-4 py-12 px-8 block border-1 w-full focusable focus:[--w-outline-offset:-2px] caret-current',
-  textArea: 'min-h-[42] sm:min-h-[45]',
-  disabled: 's-bg-disabled-subtle s-border-disabled hover:s-border-disabled! s-text-disabled pointer-events-none',
-  invalid: 's-border-negative s-text-negative! hover:s-border-negative-hover! outline-[--w-s-color-border-negative]!',
-  readOnly: 'pl-0 bg-transparent! border-0! pointer-events-none',
-  placeholder: 'placeholder:s-text-placeholder',
+  // wrapper classes
   wrapper: 'relative',
+  // input classes
+  base: 'block text-m leading-m mb-0 px-8 py-12 rounded-4 w-full focusable focus:[--w-outline-offset:-2px] caret-current', // true
+  default: 'border-1 s-text s-bg s-border hover:s-border-hover active:s-border-selected', // !isInvalid && !isDisabled && !isReadOnly
```

Comment:
@imprashast I just remembered that these classes are used as reference e.g. in the [search-newfrontier-podlet](https://github.schibsted.io/finn/search-newfrontier-podlet/blob/b1d1b5918ca79001dfc96244128a972aadd5b413/uno.config.js), which after they update the css package will break their text fields. So taking that into account, everything that happened in this PR would be a breaking change for teams that reference our classes this way.

A safe alternative here could be to create a completely new object, say `form`, and keep the old `input` object as is, just with a note that it should be removed in v2. Wdyt? Any other ideas?

Reply:
I don't think we should be concerned with this. We never recommend using our classes as shortcuts in the first place :)

Reply:
I see your point but I'm also a bit concerned about a push-back if we take a stand like that :) When in a semantic versioning world, a similar change is not treated as a patch change, which usually is associated with a safe update. We are exporting these classes from the @warp-ds/css package, which also can be understood as "I can import those", without any recommendation from our side. Now that we know there are teams using this "loophole", maybe it does make sense to allow them to safely update our packages by keeping the backwards compatibility of whatever sins we've technically allowed until v2?

Reply:
Alternatively, I can let that one team know about the change and that they should be prepared for some tweaking of their components when updating to 1.9.6. At least in the case of `input` classes it's just that one podlet.

Reply:
I talked to Vegard from the search team and he agrees that it's their risk they use our classes this way, and he appreciated the heads up that we'll release this small breaking change :)